x-pack/plugins/cases/public/components/configure_cases/connectors.tsx

```diff
@@ -101,8 +93,15 @@ const ConnectorsComponent: React.FC<Props> = ({
         description={i18n.INCIDENT_MANAGEMENT_SYSTEM_DESC}
         data-test-subj="case-connectors-form-group"
       >
-        <EuiFormRowExtended
+        <EuiFormRow
           fullWidth
+          css={css`
+            .euiFormRow__labelWrapper {
```

Comment:
nit: I am not sure the EUI team would recommend this 😄 How about styling `dropDownLabel` directly?

Reply:
fixed in https://github.com/elastic/kibana/pull/178379/commits/ac6b8d0784674a0ee251a6311f13df786f8fd06f